### PR TITLE
ROUTING-1234: having the Service as Lib example in Recorder show the …

### PR DESCRIPTION
**Pull request by juanlu-rti**
### Summary

Modify some CMake files in recording service examples so that they use the newly shipped library (7.4.0) `rtiserviceadmincpp`. Also, modified the example `service_as_lib` so that it showcases the use of the `execute_command` method to shut down the service.

### Checks

-   [x] I have updated the documentation accordingly.
-   [x] I have read the [CONTRIBUTING](https://github.com/rticommunity/rticonnextdds-examples/blob/develop/CONTRIBUTING.md) document.

### CODE REVIEW

File: examples/recording_service/service_as_lib/c++11/ServiceAsLibExample.cxx [hunk 1]
```diff
@@ -109,7 +109,24 @@ int main(int argc, char *argv[])
         embedded_service.start();
         // Wait for 'running_seconds' seconds
         std::this_thread::sleep_for(std::chrono::seconds(running_seconds));
-        embedded_service.stop();
+        // embedded_service.stop();
```
Comment (fgaranda) [inline examples/recording_service/service_as_lib/c++11/ServiceAsLibExample.cxx:112]:
```suggestion
```

Should it be removed?

**Review by fgaranda — DISMISSED** [review]

**Comment by juanlu-rti** [comment]
I have retargeted to develop like we mentioned

**Comment by juanlu-rti** [comment]
I don't understand, because I applied the feedback provided by @fgaranda and resolved the comment, yet it says that I have changes requested.
@lulivi , do you think you can review this this week?

**Comment by juanlu-rti** [comment]
@fgaranda is on vacation, I'm going to take him out of the review as he provided feedback and has been applied.

**Review by lulivi — APPROVED** [review]

**Pull request merged**